tests/mac-binding-project/MobileBinding/MobileBinding_static.csproj

```diff
@@ -9,6 +9,7 @@
     <RootNamespace>MobileBinding</RootNamespace>
     <MacResourcePrefix>Resources</MacResourcePrefix>
     <AssemblyName>MobileBinding</AssemblyName>
+    <BaseIntermediateOutputPath>..\obj\Mobile-static</BaseIntermediateOutputPath>
```

Comment:
Is this related to updating the `LangVersion`?

Reply:
hmm good point, I bet it was vsmac trying to be smart :/